engine/table/src/main/java/io/deephaven/engine/table/impl/asofjoin/RightIncrementalAsOfJoinStateManagerTypedBase.java

```diff
@@ -440,8 +436,11 @@ private int accumulateIndices(RowSet rowSet, ColumnSource<?>[] sources, IntegerA
             return 0;
         }
 
+        // store this for access by the hashing methods
+        this.hashSlots = slots;
```

Comment:
Same comment as in `buildTable`, although I only have a bug-discovery and symmetry argument here, as I think `accumulateIndices` may only be used from listeners.